package/MDAnalysis/core/universe.py

```diff
@@ -1191,7 +1226,7 @@ def add_impropers(self, values, types=None, guessed=False):
         .. versionadded:: 1.0.0
         """
         self._add_topology_objects('impropers', values, types=types,
-                                 guessed=guessed)
+                                   guessed=guessed)
```

Comment:
```suggestion
                                 guessed=guessed)
```

Revert unrelated change.